tests/test_loss.py

```diff
@@ -33,6 +33,45 @@ def test_loss_categorical():
     assert loss.item() > 4.0
 
 
+def test_loss_binary():
+    target = torch.tensor([False, True, False, True, False]).unsqueeze(1)
+    prediction = (target.float() - 0.5) * 10.0
+
+    loss_fn = PolyLoss([BinaryData()])
+    loss = loss_fn(prediction, target)
+    assert loss.item() < 0.01
+
+    # change targets
+    loss = loss_fn(prediction, ~target)
```

Comment:
I haven't come across `~` in Python code before. Interesting that pytorch seems to be using it in what is a pretty non-intuitive way (to me) - i.e., not as bit flip.

Reply:
Good point. I think I'll change this to torch.logical_not(target) so that it is more explicit what is going on.